cbmc/proofs/IotMqtt_UnsubscribeAsync/IotMqtt_UnsubscribeAsync_harness.c

```diff
@@ -0,0 +1,102 @@
+#include "iot_config.h"
+#include "private/iot_mqtt_internal.h"
+
+#include <stdlib.h>
+
+#include "mqtt_state.h"
+
+/****************************************************************/
+
+void IotListDouble_RemoveAllMatches( const IotListDouble_t * const pList,
```

Comment:
Add a comment to justify this stub as we do for the other proofs.
```c
/**
 * We abstract these functions for performance reasons. In their original
 * implementation, CBMC ended up creating byte extracts for all possible
 * objects, due to the polymorphic nature of the linked list. We assume
 * that the functions are memory safe, and we free and havoc the list to
 * demonstrate that no subsequent code makes use of the values in the list.
 */
```

Reply:
Done.